src/resources/order/models/index.model.js

```diff
@@ -69,3 +69,27 @@ export function getProducts(response) {
     return product;
   });
 }
+
+export async function fetchOrders(page, pageSize) {
+  return Order.query()
+    .withGraphFetched('user(name)')
+    .modifiers({
+      selectFirstnameAndLastname(user) {
```

Comment:
I thought this PR is aimed at users getting all their orders? If yes, why do they need to see their names?

Reply:
Oh no, this is for admin alone. that will be part of the user profile route

Reply:
Okay